SWLOR.Game.Server/Feature/AbilityDefinition/Force/ForceBurstAbilityDefinition.cs

```diff
@@ -26,53 +27,70 @@ public class ForceBurstAbilityDefinition : IAbilityListDefinition
         private static void ImpactAction(uint activator, uint target, int level, Location targetLocation)
         {
             var dmg = 0;
+            var willBonus = GetAbilityModifier(AbilityType.Willpower, activator);
 
             switch (level)
             {
                 case 1:
-                    dmg = 12;
+                    dmg = 10 + (willBonus * 2);
                     break;
                 case 2:
-                    dmg = 19;
+                    dmg = 16 + (willBonus * 4);
                     break;
                 case 3:
-                    dmg = 28;
+                    dmg = 22 + (willBonus * 6);
                     break;
                 case 4:
-                    dmg = 40;
+                    dmg = 30 + (willBonus * 8);
                     break;
             }
 
             dmg += Combat.GetAbilityDamageBonus(activator, SkillType.Force);
-
-            var attackerStat = GetAbilityScore(activator, AbilityType.Willpower);
-            var defense = Stat.GetDefense(target, CombatDamageType.Force, AbilityType.Willpower);
-            var defenderStat = GetAbilityScore(target, AbilityType.Willpower);
-            var attack = Stat.GetAttack(activator, AbilityType.Willpower, SkillType.Force);
-            var damage = Combat.CalculateDamage(
-                attack,
-                dmg, 
-                attackerStat, 
-                defense, 
-                defenderStat, 
-                0);
-            var delay = GetDistanceBetweenLocations(GetLocation(activator), targetLocation) / 18.0f + 0.35f;
-
-            AssignCommand(activator, () =>
+            var creature = GetFirstObjectInShape(Shape.Sphere, RadiusSize.Medium, GetLocation(target), true, ObjectType.Creature);
```

Comment:
The radius size on this is medium whereas the Force Lightning one is Huge. From a balance perspective should these be the same?

Reply:
The radius of Lightning is huge because it will only target up to a few enemies in that radius. Meanwhile, Burst has a smaller radius but targets everything in that radius.